var/spack/repos/builtin/packages/py-pennylane-lightning/package.py

```diff
@@ -43,8 +44,8 @@ class PyPennylaneLightning(CMakePackage, PythonExtension):
 
     # variant defined dependencies
     depends_on("blas", when="+blas")
-    depends_on("kokkos@:3.7.01", when="+kokkos")
-    depends_on("kokkos-kernels@:3.7.01", when="+kokkos")
+    depends_on("kokkos@:4.0.01", when="+kokkos")
+    depends_on("kokkos-kernels@:4.0.01", when="+kokkos")
```

Comment:
Same question, does 0.29 also support kokkos 4?

Reply:
It does.